server/model/taskModel.js

```diff
@@ -34,8 +34,9 @@ const TaskModel = {
             type:'String'
         }
     },
-    getAll: (user) => {
-        return Task.find({ createdBy: user.name }).exec();
+    getAll: (filter, skip, limit, sorter) => {
+
+        return Task.find(filter).skip(skip).limit(limit).sort(sorter).exec();
```

Comment:
Consider setting options, like in this example: https://mongoosejs.com/docs/api.html#model_Model.find

This way you can set single object and get rid of all ternary operation later, just pass something like this:

```
const options = {
  skip: params.skip
  limit: params.limit,
}
```

Reply:
Okay, so i put skip, limit and sort in options object and deleted ternary operators and now I pass filter and options to the taskModel.getAll function. Should i combine them together too ?